src/Sylius/Bundle/CoreBundle/Fixture/Factory/ShippingMethodExampleFactory.php

```diff
@@ -54,12 +58,14 @@ public function __construct(
         FactoryInterface $shippingMethodFactory,
         RepositoryInterface $zoneRepository,
         RepositoryInterface $shippingCategoryRepository,
+        RepositoryInterface $taxCategoryRepository,
```

Comment:
Why is this a BC break?

Reply:
The class is not final, so not passing this new argument could break someones application

Reply:
Why isn't it final though?

Reply:
As Mateusz said, this is BC-Break, as we are changing classes definition. If someone registered this class by his own, we would break his app. We need to add bc layer and allow a null value for this service. Therefore, it should be defined as the last one.

Reply:
Like that?

Reply:
Actually `final` does not matter here.

Adding a required argument to any public method is a BC break, as end-users could call that method without it and it will make their code crash.